sandbox/extra/src/main/java/brooklyn/entity/database/postgresql/PostgreSqlNodeSaltImpl.java

```diff
@@ -174,4 +175,8 @@ public String executeScript(String commands) {
                 ConfigBag.newInstance().configure(ExecuteScriptEffectorBody.SCRIPT, commands).getAllConfig()).getUnchecked();
     }
 
+    @Override
+    public void populateServiceNotUpDiagnostics() {
+        // FIXME no-op currently; should check ssh'able etc
+    }
```

Comment:
`FIXME` is a bit strong. This surely applies to many other entities, so highlighting this case seems a bit random ;o)